package.json

```diff
@@ -803,12 +803,17 @@
       }
     ]
   },
-  "main": "./out/client/src/extension",
+  "main": "./client/out/extension",
   "scripts": {
-    "vscode:prepublish": "npm run compile",
-    "compile": "webpack && tsc -b",
-    "watch": "tsc -b -w",
-    "test": "node ./node_modules/vscode/bin/test"
+    "vscode:prepublish": "npm run package",
+    "package": "webpack --mode production --config webpack.config.js",
+    "package:dev": "webpack --mode development --config webpack.config.js",
+    "compile": "webpack --mode none --config webpack.config.js",
+    "compile:client": "webpack --mode none --config webpack.client.config.js",
+    "compile:html_views": "webpack --mode none --config ./webpack.html_views.config.js",
+    "compile:server": "webpack --mode none --config ./webpack.server.config.js",
+    "watch": "webpack --watch --mode none --config webpack.config.js",
+    "clean": "rimraf client/out && rimraf html_views/out && rimraf server/out"
```

Comment:
This is duplicating code from the Makefile and unnecessarily adding a dependency.